slay/example_workflow/user_package/shared_processor.py

```diff
@@ -0,0 +1,20 @@
+import slay
+
+IMAGE_NUMPY = (
+    slay.Image()
+    .pip_requirements_file(
+        "/home/marius-baseten/workbench/truss/example_workflow/requirements.txt"
+    )
+    .pip_requirements(["numpy"])
+)
+
+
+class SplitText(slay.ProcessorBase):
+
+    default_config = slay.Config(image=IMAGE_NUMPY)
```

Comment:
should we change this example to pull from a serialized config instead?

Reply:
You mean having the whole config file-based?

I'll have to see what's the most important use case to support next. I think integration with existing trusses would be more useful to work on at this point than file-based "slay" config.